packages/auth/src/auth-options.ts

```diff
@@ -0,0 +1,40 @@
+import { prisma } from "@agreeto/db";
+import { PrismaAdapter } from "@next-auth/prisma-adapter";
+import type { NextAuthOptions } from "next-auth";
+import GoogleProvider from "next-auth/providers/google";
+import AzureAdProvider from "next-auth/providers/azure-ad";
+
+type GetAuthOptions = (opts: {
+  secret: string;
+  googleClientId: string;
+  googleClientSecret: string;
+  azureAdClientId: string;
+  azureAdClientSecret: string;
+  azureAdTenantId: string;
+}) => NextAuthOptions;
+
+export const getAuthOptions: GetAuthOptions = (opts) => ({
```

Comment:
i've made this a getter to try and make this package free from responsibility of providing env-vars, and put that responsibility on the app which is using the package.